tensorflow_datasets/core/features/tensor_feature.py

```diff
@@ -131,19 +132,10 @@ def encode_example(self, example_data):
                        f'For {self}')
 
     np_dtype = np.dtype(self.dtype.as_numpy_dtype)
-    if isinstance(example_data, tf.Tensor):
```

Comment:
We want to keep this check as this is a common error

Reply:
I remove this since `Tensors` are converted without any problem to `np` `arrays`. Why do we need this test?